clang/lib/AST/ASTContext.cpp

```diff
@@ -4419,6 +4419,13 @@ QualType ASTContext::getScalableVectorType(QualType EltTy, unsigned NumElts,
       EltTySize == ElBits && NumElts == (NumEls * NF) && NumFields == 1) {     \
     return SingletonId;                                                        \
   }
+#define SVE_VECTOR_TYPE_MFLOAT(Name, MangledName, Id, SingletonId, NumEls,     \
+                               ElBits, NF)                                     \
+  if (EltTy->hasIntegerRepresentation() && !EltTy->isBooleanType() &&          \
+      !EltTy->hasSignedIntegerRepresentation() && EltTySize == ElBits &&       \
+      NumElts == (NumEls * NF) && NumFields == 1) {                            \
+    return SingletonId;                                                        \
+  }
```

Comment:
This is no longer required?

Reply:
I missed that when I removed the MFLOAT.